repo/packages/M/marathon-rc/0/config.json

```diff
@@ -0,0 +1,391 @@
+          "type" : "string"
+        },
+        "assets-path" : {
+          "description" : "Set a local file system path to load assets from, instead of loading them from the packaged jar.",
+          "type" : "string"
+        },
+        "checkpoint" : {
+          "description" : "Enabled: (Default) Enable checkpointing of tasks. Requires checkpointing enabled on slaves. Allows tasks to continue running during mesos-slave restarts and upgrades Disabled: Disable checkpointing of tasks.",
+          "type" : "boolean",
+          "default" : true
+        },
+        "decline-offer-duration" : {
+          "description" : "(Default: 120 seconds) The duration (milliseconds) for which to decline offers by default",
+          "type" : "integer",
+          "default" : 120000
+        },
+        "default-accepted-resource-roles" : {
+          "description" : "Default for the defaultAcceptedResourceRoles attribute of all app definitions as a comma-separated list of strings. This defaults to all roles for which this Marathon instance is configured to receive offers.",
+          "type" : "string"
+        },
+        "disable-http" : {
+          "description" : "Disable listening for HTTP requests completely. HTTPS is unaffected.",
+          "type" : "boolean",
+          "default" : false
+        },
+        "enable-features" : {
+          "description" : "A comma-separated list of features. Available features are: vips - Enable networking VIPs UI, task_killing - Enable the optional TASK_KILLING state, available in Mesos 0.28 and later",
+          "type" : "string",
+          "default" : "vips,task_killing"
+        },
+        "env-vars-prefix" : {
+          "description" : "Prefix to use for environment variables injected automatically into all started tasks.",
+          "type" : "string"
+        },
+        "event-stream-max-outstanding-messages" : {
+          "description" : "The event stream buffers events, that are not already consumed by clients. This number defines the number of events that get buffered on the server side, before messages are dropped.",
+          "type" : "integer",
+          "default" : 50
+        },
+        "event-subscriber" : {
+          "description" : "The event subscription module to use. E.g. http_callback.",
+          "type" : "string"
+        },
+        "executor" : {
+          "description" : "Executor to use when none is specified. If not defined the Mesos command executor is used by default.",
+          "type" : "string",
+          "default" : "//cmd"
+        },
+        "failover-timeout" : {
+          "description" : "(Default: 1 week) The failover_timeout for mesos in seconds.",
+          "type" : "integer",
+          "default" : 604800
+        },
+        "framework-name" : {
+          "description" : "Framework name to register with Mesos.",
+          "type" : "string",
+          "default" : "marathon-user"
```

Comment:
Good catch. My hope was, that we can use almost the same package definition for both. Obviously this is not true.